### PR TITLE
Create page to show contact message was sent successfully

### PR DESCRIPTION
**Pull request by andela-cnwosu**
This PR:
* creates a success page for sent contact messages.
* shows an error toastr if contact details are invalid. The backend endpoint has not yet been tied to it.
* validates data on the frontend before sending the message.
* creates a separate route for all success and error messages.
* creates one template for all success and error messages.
* customizes the toastr.

screen shot 2017-05-05 at 5 09 41 pm
screen shot 2017-05-05 at 5 10 01 pm

### CODE REVIEW

File: src/js/components/partner_leads/Index.js [hunk 1]
```diff
@@ -1,7 +1,11 @@
 import React, {PropTypes} from 'react';
+import toastr from 'toastr';
 
 class PartnerLeads extends React.Component {
   render() {
+    toastr.options.positionClass = "toast-top-right";
+    toastr.options.preventDuplicates = true;
```
Comment (andela-iamadi) [inline src/js/components/partner_leads/Index.js]: Would these toastr options still work if set in `componentWillMount` ? Setting these in the render method reset it as long as re-rendering occurs. So it doesn't feel quite right.

File: src/js/components/partner_leads/eligibility_check/Index.js [hunk 2]
```diff
@@ -52,14 +53,14 @@ class EligibilityCheck extends React.Component {
       return false;
     });
     if (percentageCompletion != 100) return;
-    browserHistory.push(eligible ? '/partner-leads/apply' : '/partner-leads/ineligible');
+    browserHistory.push(eligible ? '/partner-leads/apply' : '/feedback/error/ineligible');
```
Comment (andela-iamadi) [inline src/js/components/partner_leads/eligibility_check/Index.js]: Love the `/feedback/error/ineligible` route ❤️

File: tools/README.md [hunk 3]
```diff
@@ -2,6 +2,8 @@
 
 We use feature flags to enable or disable deployed features.
 
+`NB: You must have created a .env file in the root directory.`
```
Comment (andela-iamadi) [inline tools/README.md]: Lol! thanks for adding this 😃

File: src/js/components/partner_leads/Index.js (hunk 1 above)
Comment (andela-cnwosu) [inline src/js/components/partner_leads/Index.js]: Geez. Of course. I kind of knew something was weird about this implementation. Changed it. Thanks.

**Pull request merged**